reference/forms/types/password.rst

```diff
@@ -76,4 +76,10 @@ The default value is ``''`` (the empty string).
 
 .. include:: /reference/forms/types/options/required.rst.inc
 
-.. include:: /reference/forms/types/options/trim.rst.inc
+trim
+~~~~
+**type**: ``Boolean`` **default**: ``false``
```

Comment:
Can you please add a blank line before and after this line and change `Boolean` to `bool`?